htp/htp_config.h

```diff
@@ -442,6 +442,15 @@ void htp_config_set_lzma_memlimit(htp_cfg_t *cfg, size_t memlimit);
  */
 void htp_config_set_compression_bomb_limit(htp_cfg_t *cfg, size_t bomblimit);
 
+/**
+ * Configures the maximum compression bomb time LibHTP will decompress.
+ *
+ * @param[in] cfg
+ * @param[in] seclimit
+ * @param[in] useclimit
+ */
+void htp_config_set_compression_time_limit(htp_cfg_t *cfg, size_t seclimit, size_t useclimit);
```

Comment:
there really isn't a scenario I can think of that would have > second level timeouts that make sense, so it seems we could do with a smaller type here and use it to create a local `struct timeval` in the code that uses it

Reply:
ok, enforced a hard limit of one second